### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by arunchawla-NOAA** [ufs-community/ufs-mrweather-app#142]
I found broken links for the UFS weather model when linking to read the docs manual page. I fixed those that I could find but there may be more broken links as we navigate the pages. Need to check those

**Comment by ligiabernardet** [comment]
Thanks. How did you fix that? Did you update the tag?

**Comment by arunchawla-NOAA** [comment]
It was in the wiki so I fixed there. No tag with the wiki.

**Comment by ceceliadid** [comment]
I do see a broken link in the app UG, in 1.2: "A User’s Guide for the UFS Weather Model is here" is not working. 

The wikis should be okay now.

**Comment by ligiabernardet** [comment]
I fixed this link in app branch ufs-release-v1 but I did not recreate the
tag. Waiting for more fixes to come in.

**Comment by MuayScience** [comment]
Broken link:
https://ufs-weather-model.readthedocs.io/en/release-public-v1/InputsOutputs.html

**Comment by ceceliadid** [comment]
@MuayScience Where is this reference?

**Comment by MuayScience** [comment]
@ceceliadid It is on this page:
https://ufs-mrweather-app.readthedocs.io/en/ufs-v1.0.0/inputs_outputs.html#output-files

In this section:
4.2.2. UFS Weather Model
The output files for the UFS Weather Model are described in the Users Guide.

**Comment by ligiabernardet** [comment]
I have corrected this broken link but have not recreated the tag.

**Comment by MuayScience** [comment]
It still comes up as page does not exist for me - will that only change when the tag is recreated?

**Comment by ligiabernardet** [comment]
Yes

**Comment by MuayScience** [comment]
Is it possible to reach that page some other way in the interim? I actually had a different documentation question I was trying to answer using that link when I found that it was broken.

**Comment by ligiabernardet** [comment]
https://ufs-weather-model.readthedocs.io/en/ufs-v1.0.0/InputsOutputs.html

**Comment by arunchawla-NOAA** [comment]
@ligiabernardet 
@ceceliadid 

Do we have anymore broken links? If people have checked and feel comfortable then we can close this. I have not tested since the weekend.

**Comment by ligiabernardet** [comment]
No known problems.

On March 10, 2020 7:33:11 AM MDT, arun chawla <notifications@github.com> wrote:
>@ligiabernardet 
>@ceceliadid 
>
>Do we have anymore broken links? If people have checked and feel
>comfortable then we can close this. I have not tested since the
>weekend. 
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/ufs-community/ufs-mrweather-app/issues/142#issuecomment-597087707

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by ceceliadid** [comment]
@arunchawla-NOAA @ligiabernardet  went through the App UG - broken links below. Prob good to at least fix the forum link. Have not gone through the WM UG.
1.2 (CCPP; described here) 
1.6  forum-based online support system 
3.1 The UFS MR Weather Model is itself an umbrella repository and contains a number of sub-repositories used by the model as documented here. 
4.2.2 The output files for the UFS Weather Model are described in the Users Guide.
4.3.4 The default input data for the Hurricane Dorian initialization of 08-29-2019 is also available on NOAA EMC’s FTP data repository
6. create_test is the tool that is used to test both CIME and CIME-driven models. It can be used as an easy way to run a single basic test or an entire suite of tests. create_test runs a test suite in parallel for improved performance. It is the driver behind the automated nightly testing of cime-driven models.
6.1 MACHINE This is optional; if this value is not supplied, create_test will probe the underlying machine.

**Comment by ligiabernardet** [comment]
+Laurie Carson <carson@rap.ucar.edu> Can you take care of  any doc bug
reports today while I am in this all-day retreat?

**Comment by arunchawla-NOAA** [comment]
oh we need to fix these. @ceceliadid nicely done in finding these. I hope we can check these for the model UG as well..

**Comment by ceceliadid** [comment]
Just went through the WM UG, it looks okay.

**Comment by llpcarson** [comment]
I will fix these this morning.

Laurie

**Comment by MuayScience** [comment]
@ceceliadid, @llpcarson  : About
4.2.2 The output files for the UFS Weather Model are described in the Users Guide.

This is the link I found that was supposed to be fixed when the tag was recreated. Did that not happen? Is it possible some of these have also been found and fixed but just aren't showing up yet?

**Comment by arunchawla-NOAA** [comment]
@junwang-noaa and @llpcarson in the UFS WM UG we have a link to utilities. Why specific to pyprodutil? Should jsut refer to NCEP libraries in general

https://ufs-weather-model.readthedocs.io/en/ufs-v1.0.0/CodeOverview.html#codeoverview

**Comment by junwang-noaa** [comment]
Arun,

The pyprodutil is a subcomponent under NEMS, we have
https://github.com/NOAA-EMC/NCEPLIBS-pyprodutil  listed in .gotmodules in
NEMS. No other nceplibs utility is used as submodules in ufs-weather-model.
Please let me know if you want to change the document to:

Infrastructure: Utilities

https://github.com/NOAA-EMC/NCEPLIBS
<https://github.com/NOAA-EMC/NCEPLIBS-pyprodutil>

Thanks.

Jun

**Comment by arunchawla-NOAA** [comment]
No need. This makes sense.

**Comment by llpcarson** [comment]
I've corrected the links Cecelia noted, moved the ufs-v1.0.0 tag, and recreated the Users Guide pages.  Depending on your browser, you may need to "reload" to get the updated pages.

https://ufs-mrweather-app.readthedocs.io/en/ufs-v1.0.0/index.html

Thanks, everyone, for reviewing so carefully!

**Comment by ceceliadid** [comment]
Laurie, thanks, changes look good - when you did this one you got rid of one side of a parenthesis, if it's easy to change might want to go ahead and do that.

1.2 (CCPP; described here)

**Comment by ceceliadid** [comment]
@llpcarson The only other thing I see that might be changed in the docs is in the WM UG. It's not a broken link, just something that's probably going to cause problems:
In 3.1:
A list of the bundled libraries tested with this WM release is in the top-level README of the NCEPLIBS repository (be sure to look at the tag in that repository that matches the tag on this WM release).

When you go to the link there's no README, you have to know to move the tag, so it looks at first like a mistake. So it's probably better to have the link be to:  https://github.com/NOAA-EMC/NCEPLIBS/tree/ufs-v1.0.0

Prob the same in the next paragraph for nceplibs.

And a few paragraphs down:
If your platform is not included in these platform- and compiler-specific README files, there is a more generic set of instructions in the README file at the top level of the NCEPLIBS-external repository, and at the top level of the NCEPLIBS repository. It may still be a good idea to look at some of the platform- and compiler-specific README files as a guide. Again, be sure to use the tag that corresponds to this version of the WM.

I'm not looking anymore after this!

**Comment by llpcarson** [comment]
I agree that this wording (and the link) is confusing, and if there are
other changes needed in the WM doc, we could try to clarify this.  Or, we
could wait and see what user questions (and answers) come in, and then make
an update later once we have some additional "outsider" perspective?

Let me know - I do not have permissions to update/retag in the WM repo, so
would only be able to submit a PR for Jun&others.  I can update the RTD
pages if we do re-tag.

Laurie

**Comment by ceceliadid** [comment]
I'm fine either way. I don't have other changes in the WM UG besides this, and I don't think it's too bad as it is. @arunchawla-NOAA want to make the call?

**Comment by arunchawla-NOAA** [comment]
Yes change the link for the nceplibs. Do the tag for ncepllibs-external as well so that if things change it does not mess things up. @junwang-noaa there will be a new PR coming to the WM UG, please accept and create new tag. 

Thanks!

**Comment by junwang-noaa** [comment]
The PR was merged, a new tag was created.

**Comment by llpcarson** [comment]
Thanks, Jun.  ReadTheDocs is rebuilding now, should be done in a couple of
minutes (remember to refresh/reload)

**Comment by ceceliadid** [comment]
@llpcarson are you going to fix the parenthesis typo in the app UG? is the wm UG ready? We're going live with the release on the website shortly, prob good to get this squared away...

**Comment by arunchawla-NOAA** [comment]
I am closing this
